Cybersecurity_Tools/ClipCrypt/clipcrypt/core.py

```
@@ -0,0 +1,291 @@
+        table.add_column("ID", style="cyan", no_wrap=True)
+        table.add_column("Timestamp", style="green")
+        table.add_column("Size", style="yellow")
+        table.add_column("Tags", style="magenta")
+        table.add_column("Source", style="blue")
+        
+        for entry in entries:
+            # Format timestamp
+            try:
+                dt = datetime.fromisoformat(entry['timestamp'])
+                timestamp = dt.strftime("%Y-%m-%d %H:%M")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)